src/Model/ChannelPricingLogEntry.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\PriceHistoryPlugin\Model;
+
+use Sylius\Component\Core\Model\ChannelPricingInterface;
+
+class ChannelPricingLogEntry implements ChannelPricingLogEntryInterface
+{
+    private ?int $id = null;
```

Comment:
I did it because the static analysis was crying :smile: I can suppress it but regarding [my comment](https://github.com/Sylius/PriceHistoryPlugin/pull/6#discussion_r1088681232), I think some auto-incremented field is required for logging things to have all entries always in the correct order. WDYT?

Reply:
I wonder, first of all, whether this date is not sufficient, but there is indeed a risk that it could be the same for two consecutive changes. Hence, let's base it on id by default, but still be consistent with the other plugins and not force a type for id, assuming that if someone wants to adjust it, they will be able to do it and do it correctly.

Reply:
done :+1: